packages/storefront-lwc/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.0.1",
   "description": "Sample Storefront Application with Lightning Web Components",
   "license": "MIT",
-  "type": "module",
```

Comment:
In addition to renaming those files to `.mjs` extensions, I removed the default to native modules. This means only files with `.mjs` extension will be resolved to native. That's only relevant to files in the `scripts` directory. All the core and bff code still keep `type: "module"`